lib/format.ex

```diff
@@ -2,9 +2,14 @@ defmodule Elchemy.Format do
 
   import Kernel, except: [inspect: 1, inspect: 2]
 
-  @doc "Some doc"
+  @moduledoc """
+    Formating Elixir to Elm message type.
+  """
+
   @spec inspect(term()) :: String.t()
   def inspect(term, in_type \\ false)
+
+  @doc "When term is map"
```

Comment:
A doc is only considered once for all of the function heads, so instead it should be a description of what the function does, and be located only at the top above of the first head definition (line 10)